lib/galaxy/tool_util/linters/tests.py

```diff
@@ -97,9 +97,9 @@ def lint_tsts(tool_xml, lint_ctx):
             discover_datasets = corresponding_output.find(".//discover_datasets")
             if discover_datasets is not None:
                 if output.tag == "output":
-                    if "count" not in output.attrib and output.find("./discovered_datasets") is None:
+                    if "count" not in output.attrib and output.find("./discovered_dataset") is None:
```

Comment:
I quick grep of the codebase seems to show that "discover_datasets" is the right version.

Reply:
No, `discover_datasets` is child of the `data` tag in `outputs`:

[tool > outputs > data > discover_datasets](https://docs.galaxyproject.org/en/master/dev/schema.html#tool-outputs-data-discover-datasets)

and `discovered_dataset` is the child of `output` tag is `tests`

[tool > tests > test > output > discovered_dataset](https://docs.galaxyproject.org/en/master/dev/schema.html#tool-tests-test-output-discovered-dataset)